.env

```diff
@@ -6,28 +6,28 @@
 # CHAIN_ENDPOINT=ws://localhost:9944
 # DB_PORT=5432
 
-##Interlay-Firesquid (mainnet)
-# ARCHIVE_ENDPOINT=https://interlay.archive.subsquid.io/graphql
-# CHAIN_ENDPOINT=wss://api.interlay.io/parachain
-# PROCESS_FROM=600000
-# SS58_CODEC="interlay"
-# BITCOIN_NETWORK="mainnet"
-# DB_PORT=5435
+#Interlay-Firesquid (mainnet)
```

Comment:
@bvotteler do you want to handle these changes?

@mafux777 best not to change these files.

Reply:
> @bvotteler do you want to handle these changes?

Those are from auto-formatting settings. I'm not bothered by additional spacing, or the lack of those.
Happy to deal with it if you'd prefer the spaces in the comments

Reply:
Also on the .env file? Either way, nothing to do here, I guess just concentration maybe not to touch those files in a PR for @mafux777 ?